Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -4278,6 +4279,63 @@ static bool selectionIsInsideFixedPositionContainer(Frame& frame)
     reply(pageRects.size());
 }
 
+void WebPage::drawToImage(WebCore::FrameIdentifier frameID, const PrintInfo& printInfo, size_t pageCount, CompletionHandler<void(WebKit::ShareableBitmapHandle&&)>&& reply)
```

Comment:
In reality this is *print*-to-image, right? not draw-to-image (like a software snapshot). It does printing layout and print stylesheets and whatnot?

Reply:
Ya; I named it this for consistency with `WebPage::drawToPDFiOS` which does very similar things

Reply:
Correct. I believe `drawToPDF` also suffers from this naming problem – so we could rename both (or do that one in a follow-up, if it's going to greatly increase the size of the change).